convert_state.py

```diff
@@ -1,60 +1,73 @@
+'''
+This program converts between US state names, and abbreviations. 
+For example, converting "MN" to "Minnesota" or converting "California" to "CA".
+'''
+import json
 
 
-
-import json  # import json for json processing
 def main():
+    '''
+    reads the states js and stores them as dictionary
+    '''
     with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
+        states = file.read()
+        abbreviation_to_name = json.loads(states)
 
-    print(data)
-    
     # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
+    state_abbr = abbreviation_to_name    #dictionary of state abbreviations keys and state name values 
```

Comment:
Keeping the abbreviation_to_name dictionary is better than creating a  new variable. abbreviation_to_name is a better name than state_abbr